src/core/client/framework/lib/network/createNetwork.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  authMiddleware,
+  batchMiddleware,
+  cacheMiddleware,
+  RelayNetworkLayer,
+  retryMiddleware,
+  urlMiddleware,
+} from "react-relay-network-modern/es";
```

Comment:
Why the `/es`?

Reply:
`es` means it contains es modules and have not been compiled down to  `commonjs` => allows tree shaking to do its work.